paper-badge.html

```diff
@@ -73,7 +73,7 @@
         outline: none;
       }
 
-      :host([hidden]) {
```

Comment:
No, you need both. `:host([hidden])` is so that you can apply `hidden` to the `paper-badge`, `[hidden]` is so that it can apply the attribute to its children.